region/wreckedship/main.json

```diff
@@ -1064,6 +1064,29 @@
                   ]
                 }
               ]
+            },
+            {
+              "id": 9,
```

Comment:
There could be a powered off farm at 7.  Bombs or spazer or wave.  Plasma isn't obvious how to do it.  Charge is less obvious than plasma.